### ISSUE TITLE
Transform documentation needs examples

### ISSUE THREAD

**Issue by WayneMRoseberry** [dotnet/docs#8528]
For each of the transforms, the documentation text should have an input and output example which demonstrates what the transform does.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ff8da4c6-5299-84bd-8794-c0b059d67d07
* Version Independent ID: dfe3f76f-a8d8-c996-54e5-629fa48266f4
* Content: [TermTransformColumn Class (Microsoft.ML.Legacy.Transforms)](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.legacy.transforms.termtransformcolumn?view=ml-dotnet)
* Content Source: [dotnet/xml/Microsoft.ML.Legacy.Transforms/TermTransformColumn.xml](https://github.com/dotnet/ml-api-docs/blob/live/dotnet/xml/Microsoft.ML.Legacy.Transforms/TermTransformColumn.xml)
* Product: **dotnet-ml-api**
* GitHub Login: @sfilipi
* Microsoft Alias: **johalex**

**Comment by JRAlexander** [comment]
Thanks, @WayneMRoseberry. This has been added to the backlog.
cc: @sfilipi

**Comment by sfilipi** [comment]
Yep, the new API examples do this for the transforms, you should see some of this with the 0.0.7 release.

https://github.com/dotnet/machinelearning/pull/1321
